CHANGELOG.md

```diff
@@ -2,6 +2,42 @@
 
 All notable changes to this project will be documented in this file. For future plans, see our [Roadmap](https://www.precice.org/fundamentals-roadmap.html).
 
+## 2.4.0
+
+- Added API methods for entering gradient data in the solver interface. Methods available are for scalar gradient data, vector gradient data, block scalar gradient data and block vector gradient data. (https://github.com/precice/precice/pull/1169)
+- Added CMake options to enable debug logging, trace logging and assertions in release builds. (https://github.com/precice/precice/pull/1177)
+- Added a pkg-config file for using preCICE directly from the build directory. (https://github.com/precice/precice/pull/1238)
+- Added a warning if the relative convergence measure is set too low. (https://github.com/precice/precice/pull/1266)
+- Added a warning when using `<export:vtk />` in a parallel participant. (https://github.com/precice/precice/pull/1136)
+- Added experimental API functions for waveform interpolation. Currently restricted to parallel-implicit coupling. Refer to the [user documentation](https://precice.org/couple-your-code-waveform.html). (https://github.com/precice/precice/pull/1187)
+- Added export to CSV using `<export:csv />`. (https://github.com/precice/precice/pull/1144)
+- Added export to VTP using `<export:vtp />`. (https://github.com/precice/precice/pull/1137)
+- Added export to VTU using `<export:vtu />`. (https://github.com/precice/precice/pull/1136)
+- Added the `support-radius` as an additional configuration option to the Gaussian RBF mapping configuration. (https://github.com/precice/precice/pull/1163)
+- Added the API header `precice/Tooling.hpp`, which includes utility functions. (https://github.com/precice/precice/pull/1122)
+- Added the command `check` to `binprecice`, which checks a given configuration file for correctness. (https://github.com/precice/precice/pull/1132)
+- Added the command `version` to `binprecice`, which prints the version string of the used preCICE library. (https://github.com/precice/precice/pull/1122)
```

Comment:
The renaming to `precice-tools` seems to be missing.

Reply:
It's listed in the "renaming" part :smiley:

Reply:
Not before :smirk: